docs/guides/customizing-components.stories.mdx

```diff
@@ -52,79 +53,71 @@ export const CustomRenderExample = () => (
+- customization does not depend on internal state
+- design is relatively simple
 
-##### Example: Tab component (see [Tabs](..?path=/docs/components-navigation-tabs--docs))
+[Reference implementation](https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Tabs/Tabs.tsx#L119C3-L119C38)
 
-Instead of introducing `customRenderLabel`, the `Tab` component was updated to
-allow the `label` prop to accept a `ReactNode`.
+##### Example: [Tab](../components/Tab) label
 
 ```tsx
 <Tab label={<MyCustomLabel />} />
 ```
 
-This approach was chosen to provide a simpler experience for those implementing
-Atlantis components. Customizing the `label` didn’t require exposing or
-interacting with the `Tab`’s internal state (e.g., active state styling). Once
-the design becomes more complex a custom render function would become a better
-fit.
+Customizing the `label` doesn’t require exposing or interacting with the `Tab`’s
+internal state (e.g., active state styling).
 
-#### UNSAFE\_ props
+### UNSAFE\_ props
 
-In some cases, we provide `UNSAFE_` props to allow for further advanced
-customization. These props are prefixed with `UNSAFE_` to indicate that they
-should only be used when absolutely necessary, as they may break the component's
-future compatibility.
-
-<Banner type="warning">
-  <p>
-    Use of `UNSAFE_` props is **at your own risk** and should be considered a
-    **last resort**. Future Atlantis updates may lead to unintended breakages.
-  </p>
+<Banner type="warning" dismissible={false}>
+  These properties are prefixed `UNSAFE_` for a reason! Their usage is **at your
+  own risk** and should be considered a **last resort**. Atlantis updates may
+  break these usages.
 </Banner>
 
-If you are considering using an `UNSAFE_` prop, we encourage you or your
-designers to talk to us. We may be able to suggest an alternative
-implementation, or your design can help inform future additions to Atlantis.
+`UNSAFE_` props are used to allow for advanced styling customization of:
+
+- a component's container
+- in some cases, a component's sub-components
+
+If you are considering using an `UNSAFE_` prop, we encourage you to
+[talk to us](https://getjobber.slack.com/archives/C03BJHV3PMG) to look for safer
+ways to meet your needs.
 
-- **`UNSAFE_className`**: Allows you to apply custom classnames to the
-  component's container and in some cases, sub-components.
+[Reference implementation](https://github.com/GetJobber/atlantis/blob/304e776cf7fc1b9683cecbb1d7a0d8dbebdb60bb/packages/components/src/Popover/Popover.tsx#L36-L56)
+
+##### Example: `UNSAFE_className`
 
 ```tsx
 <Component UNSAFE_className={{ container: styles.customClass }} />
 
 .customClass {
-  background-color: var(--color-surface);
-  padding: var(--space-base);
+  background-color: var(--color-surface--background);
+  padding-right: var(--space-larger);
 }
 ```
 
-```tsx
-<Component UNSAFE_className={{ container: "custom-container-class" }} />
```

Comment:
I'd like to keep this example in. It shows that `UNSAFE_className` can just be a string if needed.

Reply:
[say no more](https://github.com/GetJobber/atlantis/pull/2313/commits/50acb8dc319e3bd7d88ab0c06e0c0e9ec31b3939)